### DISCUSSION TITLE
Is there a way to turn an extension back on?

### DISCUSSION THREAD

**Discussion by XtraG** [3kh0/ext-remover#384] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
So if I used the bookmark.js code to turn a specific extension off, how could I turn it on again if you were not able to toggle off/on because of Chrome Dev Settings? Is there a code to basically reverse the effects of bookmark.js? I'm not that acquainted with this so I can't create something to reverse the effects yet.

**Comment by Aka-but-nice** [comment, marked as answer]
Go to chrome://restart
